contracts/utils/Helpers.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4 <0.9.0;
+
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+/**
+ * @dev Library of helpers for generating SQL statements from common parameters.
+ */
+library Helpers {
+    /**
+     * @notice Generates a properly formatted table name from a prefix and table id.
+     * @param prefix the user generated table prefix as a string.
+     * @param tableId the Tableland generated tableId as a uint256.
+     * @return name newly allocated string containing the table name.
+     *
+     * @dev requirements: block.chainid must refer to a supported chain.
+     */
+    function getNameFromId(string memory prefix, uint256 tableId)
+        public
+        view
+        returns (string memory)
+    {
+        return
+            string.concat(
+                prefix,
+                "_",
+                Strings.toString(block.chainid),
+                "_",
+                Strings.toString(tableId)
+            );
+    }
+
+    /**
+     * @notice Generates a CREATE statement based on a desired schema and table prefix.
+     *
+     * @param prefix the user generated table prefix as a string.
+     * @param schema a comma seperated string indicating the desired prefix. Example: "int id, text name".
+     * @return create newly allocated string containing the create statement.
+     *
+     * @dev requirements: block.chainid must refer to a supported chain.
+     */
+    function getCreateFromSchema(string memory prefix, string memory schema)
```

Comment:
how about `toCreateFromSchema`, `toInsert`, etc? in my mind, "get" involves some i/o or something. this is more like a transformation